.github/workflows/ci-components.yml

```diff
@@ -0,0 +1,89 @@
+name: CI - Components
+    
+on:
+  push:
+    branches:
+      - main
+  pull_request: {}
+
+env:
+  NODE_VERSION: '16.x'
+
+concurrency:
+   group: ci-components-${{ github.head_ref || github.ref }}
+   cancel-in-progress: true
+
+jobs:
+  test:
+    name: "Tests"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+          cache-dependency-path: yarn.lock
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+      - name: Lint
+        run: yarn run lint
+        working-directory: packages/components
+      - name: Run Tests
+        run: yarn run test:ember:percy
+        working-directory: packages/components
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_COMPONENTS }}
+
+  floating:
+    name: "Floating Dependencies"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+          cache-dependency-path: yarn.lock
+      - name: Install Dependencies
+        run: yarn install --no-lockfile --non-interactive
+      - name: Run Tests
+        run: yarn run test:ember:percy
+        working-directory: packages/components
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_COMPONENTS }}
+
+  try-scenarios:
+    name: ${{ matrix.try-scenario }}
+    runs-on: ubuntu-latest
+    needs: 'test'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        try-scenario:
+          - ember-lts-3.24
+          - ember-lts-3.28
+          - ember-release
+          - ember-beta
+         # - ember-canary
+          - ember-classic
+         # - embroider-safe
+         # - embroider-optimized
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+          cache-dependency-path: yarn.lock
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run Tests
+        working-directory: packages/components
+        run: yarn run test:ember-compatibility:ci ${{ matrix.try-scenario }}
```

Comment:
Get this error:

https://github.com/hashicorp/design-system/runs/5548203784?check_suite_focus=true#step:5:79

```
Module not found: Error: Can't resolve '@hashicorp/design-system-tokens/dist/docs/products/tokens.json' in '/tmp/broccoli-1848j7Vt67Wdm1dm/cache-244-webpack_bundler_ember_auto_import_webpack'
```

Reply:
Can see the same error locally:

design-system (components-ci) 🌔 ~ git co https://github.com/hashicorp/design-system/commit/c7459b798af0d30f08aaa1d1ca071ac087d678d6
HEAD is now at https://github.com/hashicorp/design-system/commit/c7459b798af0d30f08aaa1d1ca071ac087d678d6 Rework /components CI

design-system 🌔 ~ yarn run test:ember-compatibility

```
Module not found: Error: Can't resolve '@hashicorp/design-system-tokens/dist/docs/products/tokens.json' in '/private/var/folders/4h/xdzhnlzn31d3ts77p18y5qhm0000gq/T/broccoli-235666awj42qcgRkR/cache-244-webpack_bundler_ember_auto_import_webpack'
```

Reply:
@Dhaulagiri Seems mergeable as an incremental step forward to fix Percy and CI, outside of ember-try, but LMK what you think

From the prior PR, this commit https://github.com/hashicorp/design-system/pull/60/commits/5b1d4c04cb1cafe97684e7588b64d2fbe82dbae6 is interesting (trick Michael showed me to debug CI)

Am not sure of next steps, maybe try ember-auto-import in packages/components? 

TBH confused how https://github.com/hashicorp/design-system/blob/main/packages/components/tests/dummy/app/routes/foundations/tokens.js#L3 worked before

Reply:
Actually just thought of a better solution, separation of concerns 

- We separate out `packages/components-website` from `packages/components`
- This eliminates the need to run `ember try` 's on the dummy test addon site, which is kinda overkill anyways
- In `packages/components-website`, we can try ember-auto-import to load the tokens and components
- It's slightly more annoying to build components locally...but we could still leave a skeleton site for folks to run `ember serve` in `packages/components`
  - Could also explore in-repo hoisting to make it easier to build out `packages/components-website`

I still think this could be merged in. If we need a release, we can comment out the `TOKENS_RAW` and run the ember try's. Then I can switch gears to the alerts tmrw, but still chip away at this larger meta task?

LMK what you think